docs/templating-language.md

```diff
@@ -506,6 +507,15 @@ To query a different data center and order by shortest trip time to ourselves:
 To access map data such as `TaggedAddresses` or `Meta`, use
 [Go's text/template][text-template] map indexing.
 
+### `partitions`
+
+Query [Consul][consul] for all partitions.
+
+```golang
+{{ range partitions }}
+{{ . }}{{ end }}
+```
```

Comment:
```suggestion
{{ range partitions }}
{{ .Name }}
{{ end }}
```
```

`.` is a map. Shouldn't this be `.Name` so that the partition name is output?